ztp/gitops-subscriptions/argocd/PowerConfiguration.md

```diff
@@ -0,0 +1,224 @@
+
+Power configurations are mainly set using the `workloadHints` object in the PerformanceProfile. Detailed descriptions of each parameter can be found in the Node Tuning Operator [PerformanceProfile documentation](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_profile.md#workloadhints).
+
+## Prerequisite
+
+In order to use the power saving mode, it is required that C-states and P-states are enabled. Different vendors use varying naming for their BIOS settings, but here are some guidelines:
+
+- enable C-states:
+    - often referred to as “monitor/mwait” instructions or “C states” (or both)
+    - there may also be options to enable/disable specific C-states (e.g. C1E, C6)
+    - users may want to enable all C-states or just a subset
+- enable OS controlled P-states:
+    - this can be called “CPU power management” or “hardware P-state control”
+    - the key is to choose the option(s) that allow the OS to control P-states
+
+## Power modes
+
+### 1) [Performance mode](#performance-mode) (low latency) \[default\]
+
+Deploy the PerformanceProfile and TunedPerformancePatch policies as shown below.
+
+> **Note** that the `cpu`, `hugepages` and `realTimeKernel` (in PerformanceProfile) as well as `bootloader` (in TunedPerformancePatch) should be configured as required per the use-case. The values shown below are for illustrative purposes only.
+
+```yaml
+  sourceFiles:
+    - fileName: PerformanceProfile.yaml
+      policyName: "config-policy"
+      metadata:
+        name: openshift-node-performance-profile
+      spec:
+        cpu:
+          # These must be tailored for the specific hardware platform
+          isolated: "2-19,22-39"
+          reserved: "0-1,20-21"
+        hugepages:
+          defaultHugepagesSize: 1G
+          pages:
+            - size: 1G
+              count: 32
+        realTimeKernel:
+          enabled: false
+    - fileName: TunedPerformancePatch.yaml
+      policyName: "config-policy"
+      spec:
+        profile:
+          - name: performance-patch
+            # The 'include' line must match the PerformanceProfile metadata.name above (openshift-node-performance-${metadata.name})
+            # And the cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
+            data: |
+              [main]
+              summary=Configuration changes profile inherited from performance created tuned
+              include=openshift-node-performance-openshift-node-performance-profile
+              [bootloader]
+              cmdline_crash=nohz_full=2-19,22-39
+              [sysctl]
+              kernel.timer_migration=1
+              [service]
+              service.stalld=start,enable
```

Comment:
I don't think we should include this here - it is just going to get out of date. Perhaps you can just say that this mode is selected with the default policies?